WooCommerce/Classes/ViewRelated/Dashboard/JetpackConnectionPackageSites/JetpackBenefitItem.swift

```diff
@@ -0,0 +1,50 @@
+import SwiftUI
+
+/// Displays a Jetpack benefit with icon, title, and subtitle.
+struct JetpackBenefitItem: View {
+    let title: String
+    let subtitle: String
+    let icon: UIImage
+
+    var body: some View {
+        HStack(spacing: Layout.horizontalSpacing) {
+            Circle()
+                .frame(width: Layout.circleDimension, height: Layout.circleDimension, alignment: .center)
+                .foregroundColor(Color(.gray(.shade0)))
+                .overlay(
+                    Image(uiImage: icon)
+                        .resizable()
+                        .frame(width: Layout.iconDimension, height: Layout.iconDimension, alignment: .center)
+                )
+            VStack(alignment: .leading) {
+                Text(title).headlineStyle()
+                Spacer().frame(height: Layout.verticalTextSpacing)
```

Comment:
nit: passing spacing as `VStack` parameter will result in same layout without additional `Spacer`.

Reply:
Ooh I missed that! Updated in https://github.com/woocommerce/woocommerce-ios/pull/5379/commits/671bb4adfa3a8690e2b1c291417b3e3cb0ae3999